src/ndarray/ndarray_function.cc

```diff
@@ -38,7 +38,7 @@ void Copy<cpu, cpu>(const TBlob &from, TBlob *to,
                     RunContext ctx) {
   MSHADOW_TYPE_SWITCH(to->type_flag_, DType, {
     if (to->type_flag_ == from.type_flag_) {
-      const index_t size = from.Size();
+      const index_t size = static_cast<index_t>(from.Size());
```

Comment:
Is this needed? Size() returns a size_t and sholud be automatiocally casted

Reply:
Is automatic casting good practice ? I am not sure. Even google style guide doesn't say anything about it: https://google.github.io/styleguide/cppguide.html#Casting
I my opinion explicit casting improves code readability and helps the person reading the code understand that `Size()` doesn't return `index_t`. @apeforest  what do you think ?